### ISSUE TITLE
Sonos plugin: soco module not available on Windows

### ISSUE THREAD

**Issue by nobef** [quodlibet/quodlibet#4208]
Steps to reproduce
------------------
Such an error:
Could not find module 'soco'. You may need to install a package.

### Which version of Quod Libet?
4.5.0 

### Which operating system 
Win11

Additional Information
----------------------
[изображение](https://user-images.githubusercontent.com/77186394/203085668-f1a50d3f-2afd-4090-b425-b755af6bc849.png)

**Comment by declension** [comment]
Yes, you need to install [soco](https://pypi.org/project/soco/) to use this plugin. 

e.g. `pip install --user soco`

**Comment by bamper45** [comment]
@declension Sorry this is for windows 11

**Comment by declension** [comment]
Pip should work on Windows, but may not be accessible to the binary. 

So alternatively, we could bundle this on Windows, but it's probably not of much use to nearly all users, not sure.
